cypress/e2e/staffMembers.js

```diff
@@ -163,4 +175,148 @@ describe("Staff members", () => {
         });
     },
   );
+
+  it(
+    "should not be able to create staff member with not unique email. TC: SALEOR_3508",
+    { tags: ["@staffMembers", "@stagedOnly"] },
+    () => {
+      const firstName = faker.name.firstName();
+      const emailInvite = TEST_ADMIN_USER.email;
+      activatePlugin({ id: "mirumee.notifications.admin_email" });
+      cy.visit(urlList.staffMembers)
+        .expectSkeletonIsVisible()
+        .get(STAFF_MEMBERS_LIST.inviteStaffMemberButton)
+        .click({ force: true });
+      fillUpUserDetailsWithNotUniqueEmail(firstName, lastName, emailInvite);
+    },
+  );
+
+  it(
+    "should not be able to update staff member with not unique email. TC: SALEOR_3509",
+    { tags: ["@staffMembers", "@stagedOnly"] },
+    () => {
+      cy.visit(urlList.staffMembers)
+        .expectSkeletonIsVisible()
+        .get(SHARED_ELEMENTS.searchInput)
+        .type(`${email} {enter}`);
+      cy.waitForProgressBarToNotExist()
+        .get(STAFF_MEMBERS_LIST.staffStatusText)
+        .first()
+        .should("be.visible");
+      cy.waitForProgressBarToNotExist();
+      cy.get(STAFF_MEMBERS_LIST.staffAvatar)
+        .first()
+        .click();
+      cy.get(STAFF_MEMBER_DETAILS.staffEmail)
+        .click()
+        .clear()
+        .type(`${TEST_ADMIN_USER.email} {enter}`)
+        .get(BUTTON_SELECTORS.confirm)
+        .click()
+        .confirmationErrorMessageShouldAppear();
+    },
+  );
+
+  // Test blocked by https://github.com/saleor/saleor-dashboard/issues/2847
```

Comment:
Should we `xit` or skip this test for now as we don't know when the issue will be fixed?

Reply:
I propose `it.skip` - more verbose and way easier to guess what is the intention here even if you don't know `xit` convention.

Reply:
ok
(@michalina-graczyk )